src/main/helpers.ts

```diff
@@ -89,13 +89,9 @@ export function onNewWindowHelper(
   onBlockedExternalUrl: (url: string) => void,
   mainWindow: BrowserWindow
 ): void {
-  if (!linkIsInternal(targetUrl, urlToGo)) {
+  if (!linkIsInternal(targetUrl, urlToGo) && blockExternal) {
```

Comment:
We actually don't even use this `blockExternal` so it's safe to kill this leg of the if statement, and you can remove it as a param. It's holdover from some stuff I copied from nativefier.